internal/pkg/plan/delete-template/executor.go

```diff
@@ -0,0 +1,25 @@
+package delete_template
+
+import (
+	"context"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/state/local"
+)
+
+type executor struct {
+	*Plan
+	*local.Manager
+	ctx context.Context
+}
+
+func newExecutor(ctx context.Context, localManager *local.Manager, plan *Plan) *executor {
+	return &executor{Plan: plan, Manager: localManager, ctx: ctx}
+}
+
+func (e *executor) invoke() error {
+	uow := e.NewUnitOfWork(e.ctx)
+	for _, action := range e.actions {
+		uow.DeleteObject(action.State, action.Manifest)
+	}
+	return uow.Invoke()
```

Comment:
Ach jo a tohle taky neprojde. Všechny ty foldery ze screenshotu měly být smazaný. Ale do `e.actions` se ke smazání dostanou (to je vidět i tím že log plánu odpovídá `expected-stdout` (https://github.com/keboola/keboola-as-code/blob/a76d6861f73d0fa33a4924070393892771ef6ea0/test/cli/template-delete/complex/expected-stdout))

![CleanShot 2022-05-04 at 17 59 44@2x](https://user-images.githubusercontent.com/215660/166722407-4c66c721-9632-4699-bcc3-4d64be8da390.jpg)